src/Components/Person/Person.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+// eslint-disable-next-line
+import PropTypes from 'prop-types';
+import './Person.scss';
+
+export const Person = ({ personInfo }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = personInfo;
+
+  return (
+    // eslint-disable-next-line
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {age && (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+      )}
+
+      <p className="Person__partner">
+        {isMarried
+          ? (`My ${sex === 'f' ? ('husband') : ('wife')}'s name is ${partnerName}`)
+          : ('I am not married')}
+      </p>
```

Comment:
Don't use ternary operator inside another ternary operator
Create separate variables before return statement which should save partner sex value and marriage status info
const partnerSex = sex === 'f' ? ... : ... ;
const marriageStatus = `My ... name is ...`;

and use this variable in jsx
Such approach make your code clearly